tests/InteractsWithMessengerTest.php

```diff
@@ -45,53 +45,75 @@ final class InteractsWithMessengerTest extends WebTestCase
 
     /**
      * @test
+     * @group legacy
      */
-    public function ignore_receiver_detached_from_transport(): void
+    public function messenger_is_deprecated(): void
```

Comment:
maybe for this one we can use `$this->expectDeprecationMessage();`?

Reply:
It does not detect the deprecation message. I tested with a sub string of the original message and the complete message given by phpunit, with and without `@group legacy`. The message is never caught. Maybe I miss something again?

Reply:
never mind then, that's not really important

I must confess I've never used this method :sweat_smile: but because the name of the method was `messenger_is_deprecated` it seemed useful

Reply:
:slightly_smiling_face: